spec/controllers/v0/institution_programs_controller_spec.rb

```diff
@@ -191,8 +191,11 @@
       get(:index)
       facets = JSON.parse(response.body)['meta']['facets']
       expect(facets['provider']).not_to be_nil
-      expect(facets['provider']['acme inc']).not_to be_nil
-      expect(facets['provider']['acme inc']).to eq(1)
+      expect(facets['provider'].size).to eq(5)
+
+      acme_provider = facets['provider'].select { |provider| provider['name'] == 'ACME INC' }.first
+      expect(acme_provider['name']).to eq('ACME INC')
+      expect(acme_provider['count']).to eq(1)
```

Comment:
Suggestion:
Change `create(:institution_program, :ca_employer)` to `create_list(:institution_program, 2, :ca_employer)`
Then change these lines to: 
```ruby
acme_providers = facets['provider'].select { |provider| provider['name'] == 'ACME INC' }
expect(acme_providers.count).to eq(1)
expect(acme_providers.first['count']).to eq(2)
```
This shows that the providers are actually being grouped appropriately.
(That is, if there was a bug that caused `facets['provider']` to contain two entries for 'ACME INC', these specs would still pass, which would be bad)

Reply:
Good idea, thanks!